### ISSUE TITLE
Hi! Just A Question. 1.21

### ISSUE THREAD

**Issue by makumaku1974** [DamiansStupidCodeLMAO/1024#1]
When do you get damaged after falling into the void? Like is reinforced deepslate on top of Bedrock?

**Comment by DamiansStupidCodeLMAO** [comment]
You get damaged past -512 on the Y axis. There is no bedrock below reinforced deepslate, just void.

**Comment by makumaku1974** [comment]
Is there a way to only get the -512 Y axis separate from the reinforced deepslate?

**Comment by DamiansStupidCodeLMAO** [comment]
Yeah, sure, I'm probably going to look into making more variants for this pack in the future for situations this.

**Comment by makumaku1974** [comment]
> Yeah, sure, I'm probably going to look into making more variants for this pack in the future for situations this.

I wanted to create a structure mod which involves past forward bedrock and still keeps the -512 y. I havent seen other mods or datapack that lets you build -512 y, only yours.
